### PR TITLE
ARROW-6328: [Developer][crossbow] Click.option-s should have help text

### PR DESCRIPTION
**Pull request by ulziibay**
Click.option-s should have `help` text

## What?
Add `help` text to click.option

## Why?
Click.option should ideally have a `help` text defined to be useful.

### CODE REVIEW

**Comment by kou** [comment]
Could you open an issue on JIRA https://issues.apache.org/jira/projects/ARROW/issues and prepend "ARROW-${JIRA_ISSUE_ID}: [Developer][crossbow] " to the subject of this pull request?

https://arrow.apache.org/docs/developers/contributing.html#how-to-contribute-patches

**Comment by wesm** [comment]
This fails lint checks

**Comment by ulziibay** [comment]
@wesm it should be good to go now!

File: dev/tasks/crossbow.py [hunk 1]
```diff
@@ -724,7 +724,9 @@ def crossbow(ctx, github_token, arrow_path, queue_path):
               help='Path of changelog to update')
 @click.option('--arrow-version', '-v', default=None,
               help='Set target version explicitly')
-@click.option('--is-website', '-w', default=False)
+@click.option('--is-website', '-w', default=False, is_flag=True,
+              help='Whether to use website format for changelog. '
+                   'Uses markdown format if false.')
```
Comment (kou) [inline dev/tasks/crossbow.py]: We uses Markdown format for website too.
So the latter sentence is needless.

**Review by wesm — APPROVED** [review]
+1

**Pull request closed without merging**